webpack.config.js

```diff
@@ -37,7 +37,7 @@ const bundleEnv = config( 'env' );
 const isDevelopment = bundleEnv !== 'production';
 const shouldMinify =
 	process.env.MINIFY_JS === 'true' ||
-	( process.env.MINIFY_JS !== 'false' && bundleEnv === 'production' );
+	( process.env.MINIFY_JS !== 'false' && bundleEnv === 'production' && calypsoEnv !== 'desktop' );
```

Comment:
is the intent to turn off minification for desktop here?

Reply:
ah i see, we force MINIFY_JS true in the build-release build.

Reply:
Not exactly. The intent is to turn off minification for desktop when `MINIFY_JS=false`, even on production builds. This allows the PR builds to run without minification, giving the needed memory optimisation.

However, if you think its fine for desktop to have minification disabled even on release builds, that would make this even simpler and the macOS job could be completely removed.